test/integration/vbitcoind_node.py

```diff
@@ -147,9 +147,10 @@ def generate(self, nblocks: int, address: str = None) -> None:
                 time.sleep(tip_time - current_time)
 
     def getbestblockhash(self) -> Hexstr:
-        return self.rpc.getbestblockhash()
+        return bytes.fromhex(self.rpc.getbestblockhash())[::-1].hex()
```

Comment:
I do not think you should reverse vBTC hash in this call. You should use vBTC hashes as-is, without any modifications, unless you pass them in or out of the alt-integration library.